### DISCUSSION TITLE
Continuously console log messages

### DISCUSSION THREAD

**Discussion by remconore** [open-webui/open-webui#3838] [General]
# Bug Report

## Description

**Bug Summary:**
When logged in and inspecting the Dev tools log messages are continuously send which affect the performance.

**Steps to Reproduce:**
Log into open webui and open inspect window / Dev Tools

**Expected Behavior:**
Maybe fire once the console (log) message

**Actual Behavior:**
Continuously log messages are fired.

## Environment

- **Open WebUI Version:** 0.3.7
- **Ollama (if applicable):** 0.1.23

- **Operating System Host:** Raspberry Pi OS Bookworm Lite
- **Operating System Client:** MacOS Sonoma 14.5
- **Browser (if applicable):** Vivaldi 6.8.3381.46 (Stable channel) (arm64)

## Reproduction Details

**Confirmation:**

- [* ] I have read and followed all the instructions provided in the README.md.
- [ *] I am on the latest version of both Open WebUI and Ollama.
- [ *] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
![Screenshot 2024-07-12 at 21 04 09](https://github.com/user-attachments/assets/4ce4f41a-11d1-47c5-a4be-d0171cc1c4b7)

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

Manual Installation pip (needed pipx actually) and using nginx with reverse proxy to login on a custom domain (with SSL certificate)

## Additional Information

When asking a chat the process isn't interrupted, I receive a reply, but the console log messages continue.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by justinh-rahb** [comment]
Disable your React dev tools extension in your browser (entirely, or for your WebUI URL), simple.

**Reply by remconore** [reply]
Thank you for the quick reply. Disabling the extension solved it.
